rllib/examples/self_play_with_open_spiel.py

```diff
@@ -215,7 +235,15 @@ def policy_mapping_fn(agent_id, episode, worker, **kwargs):
 if __name__ == "__main__":
 
     args = get_cli_args()
-    ray.init(num_cpus=args.num_cpus or None, include_dashboard=False)
+    enable_new_api_stack = args.enable_new_api_stack or bool(
+        int(os.environ.get("RLLIB_ENABLE_NEW_API_STACK", 0))
+    )
+
+    ray.init(
+        num_cpus=args.num_cpus or None,
```

Comment:
oh ok, I think this is the pattern we should use for all the setup calls of all tests then